sqlx-core/Cargo.toml

```diff
@@ -16,6 +16,7 @@ migrate = ["sha2", "crc"]
 
 any = []
 
+non-zero = []
```

Comment:
A Cargo feature isn't necessary here.

It wouldn't be very useful anyways without being exposed in the `sqlx` facade.

Reply:
I removed it.